### PR TITLE
lol progressive traitor

### PR DESCRIPTION
**Pull request by StrangeWeirdKitten**
## About The Pull Request

I will consider the existence of progressive traitor again, after some gauging on staff and other maintainers. This **might** be test merged. This **might** be closed. Overall I've included some edits in the final objectives. 

- Dark Matt-eor was turned into two tunguska meteors.
- Battle Royal
- Infect AI

Battlecruiser is debatable, but might be reconsidered. Romerol is lame. We don't need traitor engineers cooking the supermatter speghetti.

## Why It's Good For The Game

I don't really scarcity. TG during our time removed Prog-Tot from sleeper agent and it's only roundstart traitor. Well, we have control of how many round start traitors can spawn.

## Proof Of Testing

It's 3 AM trust me bro (I'll test it later and most of it requires testing on live anyway)

## Changelog

:cl:
add: Progtot is back
balance: Hacking meteor satallites will instead send two tunguska meteors at the station instead of a Dark Matt-eor.
/:cl:

### CODE REVIEW

**Comment by projectkepler-ru** [comment]
augh the linter

File: modular_zubbers/code/game/gamemodes/objectives/overrides.dm [hunk 1]
```diff
@@ -10,6 +10,35 @@
...
 
-/datum/uplink_handler/generate_objectives()
-	on_update()
-	return FALSE
+/datum/traitor_objective_category/final_objective
+	name = "Final Objective"
+	objectives = list(
+//		/datum/traitor_objective/ultimate/battlecruiser = 1, Debatable.
+		/datum/traitor_objective/ultimate/battle_royale = 1,
+		/datum/traitor_objective/ultimate/dark_matteor = 1,
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/game/gamemodes/objectives/overrides.dm:18]:
```suggestion
//		/datum/traitor_objective/ultimate/dark_matteor = 1, This is broken and I need to fix it
```

**Comment by NargaInDisguise** [comment]
After having to deal with old prog-tot and now spy, we do NOT need progtot back, please god not again

**Comment by aKromatopzia** [comment]
Final objectives make a return, but the side objectives aren't seeing much change. Suppose this would make them a more credible threat, but doesn't meteors fall into the same issue as void heretic in that it kinda just causes indiscriminate annoyances?

**Comment by R3dtail** [comment]
We already struggle with spies doing anything even slightly interesting.  Even I remember how prog-tots typically rarely added much STORY  and just did what their checklists told them to do to accumulate toys.    I suppose we'll see what happens!

**Comment by AllCorruptingWyrm** [comment]
at least disable spies then.

**Comment by nikothedude** [comment]
if this is the same progtot that encourages traitors to spam side objectives for more gamer gear and sets their inevitable goal to massively disrupt the station it has the same problem progtot had on manual: obliterating the reason security would play in good faith with traitors

why release a traitor when they will inevitably just gear the hell up with tons apon tons of gamer gear? why release a traitor if they might summon a battlecruiser later? of course these are all subject to the actual environment of the game, but thats what progtot did to manuel - as heretic, progtot, etc. were added, the reason to play in good faith with antags degraded more and more until the overall atmosphere was just not conducive to it

**Comment by nikothedude** [comment]
even just one progtot has the effect of degrading the trust, and even ignoring that, the deeper issue is that it discourages traitors from actually doing creative things by dangling a carrot on a stick ahead of them if they just do repetitive tasks with no bearing on actual quality of the round

it worked great for the chaos of LRP tg, but it worked horribly for manuel (though its fine now because the playerbase shifted more towards the LRP side due to progtot and other things), and we should seriously keep that in mind

**Comment by StrangeWeirdKitten** [comment]
I am well aware of all the issues prog-tot brought, since I was one of its biggest advocates to remove it. Trust me, I know. This PR is a work in progress with diffrent things I have in mind. The more pressing issue is the server instability, which I've been taking priority.

**Pull request closed without merging**